src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -50,7 +50,8 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
     <a id="navTitle" href="/">CodeU Chat App</a>
     <a href="/conversations">Conversations</a>
       <% if (request.getSession().getAttribute("user") != null) { %>
-    <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+    <a href="/profile/<%= request.getSession().getAttribute("user") %>">
```

Comment:
nitpick: how about saving the username into a variable at the top of the page so it can be reused?

Reply:
I had actually thought about that too. Don't know why I didn't end up doing. I will correct it though.